hw/dv/tools/ralgen/ralgen.py

```diff
@@ -65,8 +65,9 @@ def main():
         cmd = os.path.join(util_path, "topgen.py")
         args = [cmd, "-r", "-o", os.getcwd(), "-t", ral_spec]
 
-    if dv_base_prefix and dv_base_prefix != "dv_base":
-        args.extend(["--dv-base-prefix", dv_base_prefix])
+    if dv_base_names and dv_base_names != ["dv_base"]:
+        args.extend(["--dv-base-names"])
+        args.extend(dv_base_names)
```

Comment:
Should this be:
```python
    if dv_base_names:
        args += ["--dv-base-names"] + dv_base_names
```
I don't think it should be "dv_base" by default.

Reply:
Oh good catch. Yeah we removed the default "dv_base" arg. Thanks and I will update that.